packages/manager/modules/sms/src/sms/batches/pending/telecom-sms-batches-pending.html

```diff
@@ -0,0 +1,112 @@
+<header>
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span data-translate="sms_batches_pending"></span>
+    </oui-back-button>
+</header>
+
+<section class="container-fluid">
+    <oui-message
+        type="error"
+        data-ng-if="$ctrl.error"
+        dismissable
+        on-dismiss="$ctrl.error = null"
+    >
+        <span
+            data-translate="sms_batches_error"
+            data-translate-values="{ 'error': $ctrl.error }"
+        ></span>
+    </oui-message>
+
+    <oui-datagrid
+        page-size="5"
```

Comment:
a lot of missing data-

Reply:
@radireddy As mentionned in several pull requests, `data-` are no longer requested on custom components like ui-kit one. If you think we should keep them, maybe we can discuss about it :)